test/JDBC/expected/TestSQLVariant-vu-verify.out

```diff
@@ -267,3 +267,22 @@ sql_variant#!#sql_variant
 2016-10-23 12:45:37.12345#!#2016-10-23 12:45:37.123456
 ~~END~~
 
+
+Select * from testsqlvariant_vu_prepare_view1;
+go
+~~START~~
+sql_variant
+abc
+~~END~~
+
+
+Select * from testsqlvariant_vu_prepare_view2;
```

Comment:
I'd prefer to put `order by` if multiple rows are expected.

Reply:
Currently the output seems consistent so I think we can skip the clause.